sdk/spring/scripts/ci_update_versions.py

```diff
@@ -0,0 +1,190 @@
+                            except AttributeError:
+                                raise ValueError('Module: {0} does not have an external dependency version.\nFile={1}\nLine={2}'.format(module_name, target_file, line))
+                        else:
+                            try:
+                                module = ext_dep_map[module_name]
+                                new_version = module.external_dependency
+                                newline = re.sub(external_dependency_version_regex, new_version, line)
+                            except AttributeError:
+                                raise ValueError('Module: {0} does not have an external dependency version.\nFile={1}\nLine={2}'.format(module_name, target_file, line))
+                    else:
+                        raise ValueError('Invalid version type: {} for module: {}.\nFile={}\nLine={}'.format(version_type, module_name, target_file, line))
+
+                    newlines.append(newline)
+                    if line != newline:
+                        file_changed = True
+                else:
+                    newlines.append(line)
+
+                if not repl_open:
+                    module_name, version_type = '', ''
+
+        if file_changed:
+            with open(target_file, 'w', encoding='utf-8') as f:
+                for line in newlines:
+                    f.write(line)
+
+    except Exception as e:
+        print("Unexpected exception: " + str(e))
+        traceback.print_exc(file=sys.stderr)
+
+def load_version_map_from_file(the_file, version_map):
+    with open(the_file) as f:
+        for raw_line in f:
+            stripped_line = raw_line.strip()
+            if not stripped_line or stripped_line.startswith('#'):
+                continue
+            module = CodeModule(stripped_line)
+            # verify no duplicate entries
+            if (module.name in version_map):
+                raise ValueError('Version file: {0} contains a duplicate entry: {1}'.format(the_file, module.name))
+            # verify that if the module is beta_ or unreleased_ that there's a matching non-beta_ or non-unreleased_ entry
+            if (module.name.startswith('beta_') or module.name.startswith('unreleased_')):
+                tempName = module.name
+                if tempName.startswith('beta_'):
+                    tempName = module.name[len('beta_'):]
+                else:
+                    tempName = module.name[len('unreleased_'):]
+                # if there isn't a non beta or unreleased entry then raise an issue
+                if tempName not in version_map:
+                    raise ValueError('Version file: {0} does not contain a non-beta or non-unreleased entry for beta_/unreleased_ library: {1}'.format(the_file, module.name))
+
+            version_map[module.name] = module
+
+def display_version_info(version_map):
+    for value in version_map.values():
+        print(value)
```

Comment:
Could you please use log instead of `print`?
Refer to `get_spring_boot_managed_external_dependencies.py`:

```
    parser.add_argument(
        '--log',
        type=str,
        choices=['debug', 'info', 'warn', 'error', 'none'],
        required=False,
        default='info',
        help='Set log level.'
    )
    args = parser.parse_args()
    log_dict = {
        'debug': Log.DEBUG,
        'info': Log.INFO,
        'warn': Log.WARN,
        'error': Log.ERROR,
        'none': Log.NONE
    }
    log.set_log_level(log_dict[args.log])
    # log.log_level_test()
```